### ISSUE TITLE
add NCVISUAL_OPTION_VERTALIGNED

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#1461]
We should make it easier to vertically align images along the top, center, or bottom of their plane. Add `NCVISUAL_OPTION_VERALIGNED` to interpret `y` as a new set of `enum`s:

```c
NCVALIGN_NONE,
NCVALIGN_TOP,
NCVALIGN_CENTER,
NCVALIGN_BOTTOM
```

**Comment by dankamongmen** [comment]
Seems to be working!

**Comment by dankamongmen** [comment]
Solved in #1467 .
